strapi/strapi
# Add confirmation to continue when assets backup folder cannot be created

### What does it do?
It adds a confirmation layer to avoid having an error when the assets backup folder cannot be created during the data transfer process. Now using the `--force` flag skips that error in case it happens (probably due to permission issues)

### Why is it needed?
To let the decide if they want to stop the process in case there is an error in the assets backup folder creation process or if they prefer to continue without the backup folder assuming the consequences. 

### How to test it?
Using the monorepo, duplicate the getstarted process and change its name in the package.json file to avoid errors. Then go to this file https://github.com/strapi/strapi/blob/ca0a7f5fcb2ea92722dc7922d696c9b2654ffa99/packages/core/data-transfer/src/strapi/providers/local-destination/index.ts#L195 and throw an error inside the try. Check that in the console the confirmation message gets prompted. You can also check that without throwing the error everything works normally and using the `--force` flag it skips confirmation and still doesn't throw the error. 

### Related issue(s)/PR(s)
https://github.com/strapi/strapi/issues/16340
